modules/infinite-scroll/infinity.php

```diff
@@ -881,6 +881,8 @@ function action_wp_footer_settings() {
 			'query_before'    => current_time( 'mysql' ),
 			'last_post_date'  => self::get_last_post_date(),
 			'body_class'	  => self::body_class(),
+			'loading_text'	  => esc_js( __( 'Loading', 'jetpack' ) ),
+			'loaded_text'	  => esc_js( __( 'Loaded new posts', 'jetpack' ) ),
```

Comment:
Could these include the page number loaded? Otherwise you might just hear "loading" with no context.

Reply:
I will see if I can make that happen, at the moment these translations are passed into the page once.

Reply:
As the translations are statically coded, I went with dropping the loaded stat and a more verbose loading message.